docs/modules/ROOT/pages/overview.adoc

```diff
@@ -1,14 +1,97 @@
 = Hazelcast Command-Line Client (CLC)
-:description: Short description with main points
+:url-github-clc: https://github.com/hazelcast/hazelcast-cloud-cli/blob/master/README.md 
+:description: You can use Hazelcast Command Line Client (CLC) to connect to and interact with clusters on Hazelcast Platform and Hazelcast Viridian, direct from the command line or through scripts.
 
-What does the product do?
-what are some examples of where you’d use it?
-Who is it for?
-What problem does the product solve?
-What modes are available and what are they?
-What (if any) limitations does the product have?
-How do you get started?
+{description}
+
+Hazelcast CLC simplifies the client setup process for Unix-like local environments. Within minutes of installation, you can start to perform common tasks on clusters.
+
+== Use Cases
+
+Example use cases for Hazelcast CLC.
+
+=== Basic Cluster Management for Hazelcast Platform
+
+Perform xref:hzc-cluster.adoc[administration tasks] on multiple clusters without writing code. Instead, use <<interactive-mode, interactive mode>> to explore available commands.
+
+Check or change the state of a cluster, update a configuration option, then switch to another cluster direct from the command line. For maximum flexibility, you can set cluster connection details at the command line or using a xref:configuration.adoc[custom configuration file]. 
+
+NOTE: {hazelcast-cloud} has a separate link:{url-github-clc}[CLI for cluster management].
+
+=== Access Data for Debugging
+
+Get xref:hzc-map.adoc[direct access to map data] for quick debugging of prototypes or production applications. Easily read or query data on a cluster using <<sql-mode, SQL mode>> to view query results, and then perform updates without changing tools.
+
+=== Scripting for Automation
+
+Work in <<non-interactive-mode, non-interactive mode>> to script and execute repetitive administration, integration, or testing tasks.
+
+== Modes
+
+Hazelcast CLC works in three modes:
+- Non-Interactive
+- Interactive
+- SQL
+
+You can easily switch between modes to suit the context in which you're working.
+// Is this true, can you easily disable interactive mode? I couldn't. 
```

Comment:
I assume you just use `exit` to get out of interactive mode. @utku-caglayan

Reply:
Yes, from interactive mode one can exit by typing "exit" or pressing Ctrl+c together. Also you can switch to SQL Browser by typing "sql" from the interactive mode.

Reply:
I think this is related to an issue #96

Reply:
I have added the ways to exit this mode and will track #96.

Reply:
Oh I see, sorry to hear :(. I will try to reproduce the mentioned issue. Thanks